api/StargateRESTv2.yaml

```diff
@@ -1,24 +1,33 @@
 openapi: 3.0.1
```

Comment:
This needs updated for the new index related endpoints. Do you want me to make a branch off of this or wait until after you merge?

Reply:
I added the new index related endpoints. Not completely happy with the Response example for GET ALL INDEXES, but it is sufficient.